src/hooks/chatSidebar.ts

```diff
@@ -0,0 +1,85 @@
+import { useCallback } from "react";
+
+import {
+  chatVisibilitySelector,
+  selectedChatSettingsSelector,
+} from "utils/selectors";
+
+import {
+  setPrivateChatTabOpened,
+  setVenueChatTabOpened,
+  setChatSidebarVisibility,
+} from "store/actions/Chat";
+
+import { useSelector } from "./useSelector";
+import { useDispatch } from "./useDispatch";
+import { useNumberOfUnreadChats } from "./privateChats";
+import { useVenueId } from "./useVenueId";
+import { useConnectCurrentVenueNG } from "./useConnectCurrentVenueNG";
+
+export const useChatSidebarControls = () => {
+  const dispatch = useDispatch();
+  const isExpanded = useSelector(chatVisibilitySelector);
+  const chatSettings = useSelector(selectedChatSettingsSelector);
+
+  const expandSidebar = useCallback(() => {
+    dispatch(setChatSidebarVisibility(true));
+  }, [dispatch]);
+
+  const collapseSidebar = useCallback(() => {
+    dispatch(setChatSidebarVisibility(false));
+  }, [dispatch]);
+
+  const toggleSidebar = useCallback(() => {
+    if (isExpanded) {
+      collapseSidebar();
+    } else {
+      expandSidebar();
+    }
+  }, [expandSidebar, collapseSidebar, isExpanded]);
+
+  const selectVenueChat = useCallback(() => {
+    expandSidebar();
+    dispatch(setVenueChatTabOpened());
+  }, [dispatch, expandSidebar]);
+
+  const selectPrivateChat = useCallback(() => {
+    expandSidebar();
+    dispatch(setPrivateChatTabOpened());
+  }, [dispatch, expandSidebar]);
+
+  const selectRecipientChat = useCallback(
+    (recipientId: string) => {
+      expandSidebar();
+      dispatch(setPrivateChatTabOpened(recipientId));
+    },
+    [dispatch, expandSidebar]
+  );
```

Comment:
Instead of dispatching 2 different redux actions for each of these, we can handle the 'expand sidebar' part of this at the reducer level, within handling the `setPrivateChatTabOpened` action.